examples/jsm/renderers/nodes/inputs/Matrix4Node.js

```diff
@@ -0,0 +1,18 @@
+import InputNode from '../core/InputNode.js';
+import { Matrix4 } from '../../../../../build/three.module.js';
+
+class Matrix4Node extends InputNode {
+
+	constructor( value = new Matrix4() ) {
```

Comment:
Is there a reasons for providing a default parameter in this class but not in `Vector3Node`?

Reply:
For `CameraNode` I use `Vector3` value of  `Vector3Node` in `Matrix4Node` I replace the value
https://github.com/mrdoob/three.js/blob/d0ca03dc5c3060182693ca2c53b0c1a1005a4ff3/examples/jsm/renderers/nodes/accessors/CameraNode.js#L54
https://github.com/mrdoob/three.js/blob/d0ca03dc5c3060182693ca2c53b0c1a1005a4ff3/examples/jsm/renderers/nodes/accessors/CameraNode.js#L50

Reply:
Okay. I could imagine that this might confuse developers. Maybe better to use a consistent policy in these classes.

Reply:
Hmm... but all InputNode extended classe have a default https://github.com/mrdoob/three.js/blob/e432081bc1adbff01c8ad7ba32eeb37c89a74eb7/examples/jsm/renderers/nodes/inputs/Vector3Node.js#L4-L6

Reply:
Oh sorry. I've checked the version on my `dev` branch. I've seen you updated the code 👍 .